guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+If there are any released {ProjectServer} updates, those will be included when performing this update.
+We do not recommend updating the operating system independently of {ProjectServer}.
```

Comment:
Title: Updating the operating system 
Text: We do not recommend updating the operating system 

Yes, that's not what you mean, but it can be read that way.

We should somehow emphasize that you should not update OS alone IF there are project updates. But not sure how to w/o making the sentence super long :/

Reply:
I want to emphasize rather that you *cannot* update just the OS if there are pending project updates

Reply:
> Title: Updating the operating system Text: We do not recommend updating the operating system
> 
> Yes, that's not what you mean, but it can be read that way.

Yep, that's how I read it.

> We should somehow emphasize that you should not update OS alone IF there are project updates. But not sure how to w/o making the sentence super long :/

There might be a prerequisite in here somewhere: "Check that there are no project updates. <instructions on how to check> If there are,  do not update OS. <do XYZ instead>"